packages/tokamak/sdk/tasks/deposit-withdraw-eth.ts

```diff
@@ -0,0 +1,264 @@
+    CanonicalTransactionChain: zeroAddr,
+    BondManager: zeroAddr,
+    AddressManager: addressManager,
+    L1CrossDomainMessenger: l1CrossDomainMessenger,
+    L1StandardBridge: l1StandardBridge,
+    OptimismPortal: optimismPortal,
+    L2OutputOracle: l2OutputOracle,
+  }
+  console.log('l1 contracts:', l1Contracts)
+
+  const bridges = {
+    TON: {
+      l1Bridge: l1Contracts.L1StandardBridge,
+      l2Bridge: predeploys.L2StandardBridge,
+      Adapter: TONBridgeAdapter,
+    },
+  }
+
+  const l1ChainId = (await l1Provider.getNetwork()).chainId
+  const l2ChainId = (await l2Provider.getNetwork()).chainId
+
+  const messenger = new CrossChainMessenger({
+    bedrock: true,
+    contracts: {
+      l1: l1Contracts,
+    },
+    bridges,
+    l1ChainId,
+    l2ChainId,
+    l1SignerOrProvider: l1Wallet,
+    l2SignerOrProvider: l2Wallet,
+  })
+
+  let l1Balance = await l1Wallet.getBalance()
+  console.log('l1 eth balance: ', l1Balance.toString())
+
+  let l2Balance = await ethContract.balanceOf(l2Wallet.address)
+  console.log('l2 eth balance:', l2Balance.toString())
+
+  const depositTx = await messenger.depositETH(amount)
+  await depositTx.wait()
+  console.log('depositTx:', depositTx.hash)
+
+  await messenger.waitForMessageStatus(depositTx.hash, MessageStatus.RELAYED)
+
+  l1Balance = await l1Wallet.getBalance()
+  console.log('l1 eth balance: ', l1Balance.toString())
+
+  l2Balance = await ethContract.balanceOf(l2Wallet.address)
+  console.log('l2 eth balance:', l2Balance.toString())
+}
+
+const withdrawETH = async (amount: NumberLike) => {
+  console.log('Withdraw ETH:', amount)
+  console.log('l1 address:', l1Wallet.address)
+  console.log('l2 address:', l2Wallet.address)
+
+  const ethContract = new ethers.Contract(predeploys.ETH, erc20ABI, l2Wallet)
+
+  const l1Contracts = {
+    StateCommitmentChain: zeroAddr,
+    CanonicalTransactionChain: zeroAddr,
+    BondManager: zeroAddr,
+    AddressManager: addressManager,
+    L1CrossDomainMessenger: l1CrossDomainMessenger,
+    L1StandardBridge: l1StandardBridge,
+    OptimismPortal: optimismPortal,
+    L2OutputOracle: l2OutputOracle,
+  }
+
+  const bridges = {
+    TON: {
+      l1Bridge: l1Contracts.L1StandardBridge,
+      l2Bridge: predeploys.L2StandardBridge,
+      Adapter: TONBridgeAdapter,
+    },
+  }
+
+  const l1ChainId = (await l1Provider.getNetwork()).chainId
+  const l2ChainId = (await l2Provider.getNetwork()).chainId
+
+  const messenger = new CrossChainMessenger({
+    bedrock: true,
+    contracts: {
+      l1: l1Contracts,
+    },
+    bridges,
+    l1ChainId,
+    l2ChainId,
+    l1SignerOrProvider: l1Wallet,
+    l2SignerOrProvider: l2Wallet,
+  })
+
+  console.log('l1 eth balance: ', (await l1Wallet.getBalance()).toString())
+
+  let l2Balance = await ethContract.balanceOf(l2Wallet.address)
+  console.log('l2 eth balance: ', l2Balance.toString())
+
+  const withdraw = await messenger.withdrawERC20(ETH, predeploys.ETH, amount)
```

Comment:
```suggestion
  const withdraw = await messenger.withdrawETH(amount)
```

Reply:
In L2, ETH is an ERC20 token, so we use withdrawERC20 instead of withdrawETH

Reply:
@nguyenzung Yes, you can.
When I commented this line, I updated `withdrawETH` with predeploys.ETH and zeroAddress as the params in my PR, so it is compatible with your changes
<img width="914" alt="Screenshot 2024-02-05 at 10 42 07" src="https://github.com/tokamak-network/tokamak-titan-canyon/assets/149703736/493573fb-5325-43de-8aad-052612475e40">


You can keep you code